drivers/clock_control/clock_control_mcux_ccm.c

```diff
@@ -89,6 +89,12 @@ static int mcux_ccm_get_subsys_rate(struct device *dev,
 				(CLOCK_GetDiv(kCLOCK_Usdhc2Div) + 1U);
 		break;
 #endif
+
+
+
+
+
+
```

Comment:
why are you adding blank lines?

Reply:
> Please make the clock control changes in two separate commits - 1 commit for the whitespace changes, 1 commit to add the EDMA clock.

sorry this is not the way to add blank lines?

Reply:
No, I was referring to formatting changes, like adding spaces or tabs to fix alignment. It's easier to review when nonfunctional changes like this are in a separate commit.

Reply:
I see

Reply:
Thanks, it looks good